src/core/ext/transport/chttp2/transport/frame_settings.h

```diff
@@ -87,7 +87,7 @@ extern const grpc_chttp2_setting_parameters
     grpc_chttp2_settings_parameters[GRPC_CHTTP2_NUM_SETTINGS];
 
 /* Create a settings frame by diffing old & new, and updating old to be new */
-grpc_slice grpc_chttp2_settings_create(uint32_t *old, const uint32_t *new,
+grpc_slice grpc_chttp2_settings_create(uint32_t *old, const uint32_t *newval,
```

Comment:
It is... we need to include this header in a C++ file... new isn't acceptable. It could be a separate PR, but PR submission overhead is sufficiently high that I don't want to take the pain.